3_visualize/src/visualize_data.R

```diff
@@ -0,0 +1,73 @@
+############## Visualize data  ######################
+source("2_process/src/process_data.R") 
```

Comment:
this `source()` call shouldn't be in this script. all the `source`ing should happen in README.R (and in fact pretty son we're going to take out all the `source`ing everywhere, once we get to using **scipiper** instead of README.R to connect your pipeline)